package.json

```diff
@@ -13,7 +14,7 @@
     "styled-components": "^4.3.2"
   },
   "scripts": {
-    "start": "REACT_APP_API_URL=http://localhost:9000 react-scripts start",
```

Comment:
We actually `REACT_APP_API_URL=http://localhost:9000` as part of the `start` script to get the correct API_URL for our path in all of our request handlers.

Please make sure this gets reverted.